public/pages/forgot-password.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
The css stylings for email templates have to be inlined for it to work, moving the css to a new file does not work well.